resource-management/docs/performance/perf-results.md

```diff
@@ -0,0 +1,98 @@
+## Table of Contents
+- [Test Environment Configurations](#test-environment)
+- [Performance Result for Release 0.2.0](#performance-result-for-release-020)
+- [Performance Result for Release 0.1.0](#performance-result-for-release-010)
+
+## Test Environment
+
+### Test Config
+* 5 regions in GCE cross US continent, premium network
+* Each region uses one simulator to simulate 40 clusters; each cluster has 25K nodes, total 1M nodes per region.
+* 100 schedulers (clients of GRS) each request 50K nodes.
+
+### Test hosts configuration
+* GRS service: n1-standard-32 VM with 32 core and 120GB Ram, 500GB SSD,
+* Scheduler and simulators: n1-standard-8 VMs
+
+### Services locations
+* GRS Service:
+
+|        Region |             Location |
+|--------------:|---------------------:|
+| us Central-1a | Council bluffs, IOWA |
+
+* Simulators:
+
+|        Region |             Location |
+|--------------:|---------------------:|
+| us Central-1a | Council bluffs, IOWA |
+|    us east1-b |    Moncks COrner, SC |
+|    us west2-a |               LA, CA |
+|    us west3-c | Salt Lake city, Utah |
+|    us west4-a |    las Vegas, Nevada |
+
+* Schedulers:
+
+|     Region |             Location |
+|-----------:|---------------------:|
+| us west3-b | Salt Lake city, Utah |
+| us east4-b |    Ashburn, Virginia |
```

Comment:
for 5M official test, we have 5 simulator: us-west3-b,us-west3-b,us-central1-b,us-east1-c,us-central1-a

Reply:
us-west3-b twice?

Reply:
./docs/setup-guide/setup-grs-test.md:export SERVER_ZONE=us-central1-a SIM_ZONE=us-central1-a,us-east1-b,us-west2-a,us-west4-a,us-west3-c CLIENT_ZONE=us-west3-b,us-east4-b

Can this be more specific?